src/pipecat/services/inworld/tts.py

```diff
@@ -0,0 +1,600 @@
+                      - True: Real-time audio chunks as they're generated (lower latency)
+                      - False: Complete audio file generated first, then chunked for playback (simpler)
+            base_url: Base URL for Inworld HTTP API. If None, automatically selected based on streaming mode:
+                     - Streaming: "https://api.inworld.ai/tts/v1/voice:stream"
+                     - Non-streaming: "https://api.inworld.ai/tts/v1/voice"
+                     Should normally not be changed unless using a different environment.
+            sample_rate: Audio sample rate in Hz. If None, uses default from StartFrame.
+                        Common values: 48000 (high quality), 24000 (good quality), 16000 (basic)
+            encoding: Audio encoding format. Supported options:
+                     - "LINEAR16" (default) - Uncompressed PCM, best quality
+                     - Other formats as supported by Inworld API
+            params: Input parameters for voice and model configuration. Use this to specify:
+                   - voice_id: Voice selection ("Ashley", "Hades", etc.)
+                   - model: TTS model ("inworld-tts-1", "inworld-tts-1-max", etc.)
+                   - temperature: Voice temperature control for variability (range: [0, 2], e.g., 0.8, optional)
+                   If None, uses default values (Ashley voice, inworld-tts-1 model).
+                   Note: Language is automatically inferred from input text.
+            **kwargs: Additional arguments passed to the parent TTSService class.
+
+        Note:
+            The aiohttp_session parameter is required because Inworld's HTTP API
+            benefits from connection reuse and proper async session management.
+        """
+        # Initialize parent TTSService with audio configuration
+        super().__init__(sample_rate=sample_rate, **kwargs)
+
+        # Use provided params or create default configuration
+        params = params or InworldTTSService.InputParams()
+
+        # Store core configuration for API requests
+        self._api_key = api_key  # Authentication credentials
+        self._session = aiohttp_session  # HTTP session for requests
+        self._streaming = streaming  # Streaming mode selection
+
+        # Set base URL based on streaming mode if not provided
+        if base_url is None:
+            if streaming:
+                base_url = "https://api.inworld.ai/tts/v1/voice:stream"  # Streaming endpoint
+            else:
+                base_url = "https://api.inworld.ai/tts/v1/voice"  # Non-streaming endpoint
+        self._base_url = base_url  # API endpoint URL
+
+        # Build settings dictionary that matches Inworld's API expectations
+        # This will be sent as JSON payload in each TTS request
+        # Note: Language is automatically inferred from text by Inworld's models
+        self._settings = {
+            "voiceId": params.voice_id or "Ashley",  # Voice selection from params
```

Comment:
Since you're already setting defaults above, I think this can be simplified:
```suggestion
            "voiceId": params.voice_id,  # Voice selection from params
```